### ISSUE TITLE
file attachment broke

### ISSUE THREAD

**Issue by wanderer** [ssbc/patchwork#368]
whenever i try to attach a file I get
```
TypeError: Cannot read property 'ssb' of undefined
    at FileReader.reader.onload (http://localhost:7777/main.build.js:28087:23)
```
I'm using latest on master

**Comment by pfraze** [comment]
Thanks for the heads up, should be an easy fix

**Comment by pfraze** [comment]
fixed in patchkit-post-composer@1.0.2
